frontend/src/components/tournament/ReorderTeams.js

```diff
@@ -0,0 +1,92 @@
+import { useDragDropContext } from "@thisbeyond/solid-dnd";
+import {
+  DragDropProvider,
+  DragDropSensors,
+  DragOverlay,
+  SortableProvider,
+  createSortable,
+  closestCenter
+} from "@thisbeyond/solid-dnd";
+import { createSignal, For } from "solid-js";
+
+const TeamInfo = props => (
+  <div class="flex flex-row p-2">
+    <div class="basis-3/4 text-start">{props.teamName}</div>
+    <div class="basis-1/4 dark:bg-gray-600/50 bg-gray-100 text-center rounded-xl">
+      {props.teamId}
+    </div>
+  </div>
+);
+
+const Sortable = props => {
+  const sortable = createSortable(props.teamId);
+  const [state] = useDragDropContext();
+  return (
+    <div
+      use:sortable
+      class="sortable text-gray-500 dark:text-gray-400 m-2 rounded-lg bg-gray-200 border-gray-300 border-b dark:bg-gray-800 dark:border-gray-700"
+      classList={{
+        "opacity-25": sortable.isActiveDraggable,
+        "transition-transform": !!state.active.draggable
+      }}
+    >
+      <TeamInfo teamId={props.teamId} teamName={props.teamsMap[props.teamId]} />
+    </div>
+  );
+};
+
+/**
+ * @param {object} props
+ * @param {int[]} props.teams
+ * @param {callback} props.updateTeamSeeding
+ */
+const ReorderTeams = props => {
+  const [activeItem, setActiveItem] = createSignal(null);
+
+  const onDragStart = ({ draggable }) => setActiveItem(draggable.id);
+
+  const onDragEnd = ({ draggable, droppable }) => {
+    if (draggable && droppable) {
+      const currentItems = props.teams;
+      const fromIndex = currentItems.indexOf(draggable.id);
+      const toIndex = currentItems.indexOf(droppable.id);
+      if (fromIndex !== toIndex) {
+        const updatedItems = currentItems.slice();
+        updatedItems.splice(toIndex, 0, ...updatedItems.splice(fromIndex, 1));
```

Comment:
Here cant we run a loop through updatedItems and set teamSeed field in each element to the index of the element and display the teamSeed in the TeamInfo?